CourseSearchBuilderBlobTrigger/search.py

```diff
@@ -172,3 +183,140 @@ def bulk_create_courses(self, documents):
             )
 
             raise exceptions.StopEtlPipelineErrorException()
+
+
+class Synonym:
+    """Creates a new synonym"""
+
+    def __init__(self, url, api_key, api_version):
+
+        self.query_string = "?api-version=" + api_version
+        self.synonym_name = "english-course-title"
+        self.url = url
+
+        self.headers = {"Content-Type": "application/json", "api-key": api_key}
+
+    def delete_if_already_exists(self):
```

Comment:
The only call I can see to this method is commented out?

Reply:
You are correct, I forgot to clean this up once I had got it working. Thanks